src/content/components/landing-page/pedagogical-tool/PedagogicalTool.astro

```diff
@@ -0,0 +1,72 @@
+---
+import claseImage from "../../../../assets/clase.jpeg";
+import { Image } from "astro:assets";
+---
+
+<div id="pedag-container">
+  <Image class="class-img" src={claseImage} alt="Wollok en la clase" />
+  <div class="description">
+    <h2 class="title">Una herramienta con sentido pedagógico</h2>
+    <div class="bullet-points">
+      <p>
+        Su lenguaje permite incorporar los conceptos principales del paradigma en
+        forma incremetal.
+      </p>
+      <p>
+        Lográ programas con polimorfismo desde el primer día. Incrementa gradualmente
+        la complejidad con clases y mixins. Llega a dominar las colecciones usando
+        bloques de código (closures).
```

Comment:
```suggestion
        Lográ programas con polimorfismo desde el primer día. Incrementá gradualmente
        la complejidad con clases y mixins. Llegá a dominar las colecciones usando
        bloques de código como parámetros.
```

Reply:
Después habría que ver si queremos marcar con negrita o itálica palabras como `polimorfismo` o `mixines` (?